cmd/relayer/validate.go

```diff
@@ -0,0 +1,156 @@
+package main
```

Comment:
This doesn't seem like the right package for this.  Maybe we should create a hierarchy under the top level `relayer` package to separate server and client code? Code in `cmd/PROG_NAME` is for the command line shell. Parsing the command line, setting up the signal handler, reading config files, etc. are all in scope, but it is generally not where you would expect to find the entire program's implementation.

Reply:
I'd argue since the code is only used by the relayer server and nothing else, it's fine to be here, but I can move it out to a different package if you'd like. the "validation func" is also passed into the constructor for the relayer, so I'd say it's part of setting up the relayer server

Reply:
Okay. I like the way that it was broken off into a separate file. If we do ever move it somewhere else in the future, that will make it a lot easier since the code and tests are separate from `main.go`..